src/main/java/org/springframework/data/repository/core/NamedQueries.java

```diff
@@ -37,5 +39,6 @@
 	 * @param queryName
 	 * @return
 	 */
+	@Nullable
 	String getQuery(String queryName);
```

Comment:
Would it make sense to enforce non-null here because `NamedQueries` exposes a `hasQuery(…)` method?